python/cudf/cudf/tests/test_series.py

```diff
@@ -573,6 +573,29 @@ def test_series_value_counts(dropna, normalize):
         assert_eq(expect, got, check_dtype=False, check_index_type=False)
 
 
+@pytest.mark.parametrize("bins", [1, 2, 3])
+def test_series_value_counts_bins(bins):
+    psr = pd.Series([1.0, 2.0, 2.0, 3.0, 3.0, 3.0])
+    gsr = cudf.from_pandas(psr)
+
+    expected = psr.value_counts(bins=bins)
+    got = gsr.value_counts(bins=bins)
+
+    assert_eq(expected, got, check_dtype=False)
```

Comment:
```suggestion
    assert_eq(expected.sort_index(), got.sort_index(), check_dtype=False)
```

Reply:
OK cool so you just want to resolve the issue Marlene raised by testing the sorted outputs. That seems fine to me.